contracts/Enforcer.sol

```diff
@@ -86,8 +86,7 @@ contract Enforcer {
 
     // receive result from Verifier contract
     function result(bytes32 executionId, bool solverWon, address challenger) public onlyVerifier() {
-        require(executions[executionId].startBlock != 0);
-        require(executions[executionId].startBlock + challengePeriod > block.number);
```

Comment:
This line must be remove for `claimTimeout` to work, isn't it?

Reply:
I think this check should stay. If it fails in your unit tests, try to change challengePeriod.

For reference, you can check https://github.com/leapdao/solEVM-enforcer/pull/66/files#diff-b7b64c17155625a2b3884c0b7b50a72dR98

Reply:
I'm confused about `timeout` in **Verifier** and `challengePeriod` in **Enforcer**.
So a new challenge can be put up as long as it's within `challengePeriod`.
Then, the new challenge should be done within `timeout` by the **Verifier**.
But there is no connection between the two, so a challenge can be finished within `timeout` by **Verifier** but the challenge is no longer valid according to the **Enforcer**.
I think we should do something here.

Reply:
Ya, there is a problem that if you challenge too late, you will reach the challenge period.
We discussed that in a meeting and that is something the 'tools' who do the part later needs to care about. Because we still need to enforce a 'period' in which can be challenged.
And because the challengePeriod will be 3 days as the first setup (the exitHandler stuff) this isn't a big problem.

Reply:
I've recovered that line